website/screens/components/footer/specs/FooterSpecsPage.tsx

```diff
@@ -272,6 +324,27 @@ const sections = [
           </DxcTable>
         ),
       },
+      {
+        title: "Height (reduced)",
```

Comment:
I would include everything in a `Height` section, I mean:

Height:

- Default:
     (table with default tokens)
- Reduced:
    (table with reduced tokens)